### ISSUE TITLE
geoserver datadir overwriten in docker 

### ISSUE THREAD

**Issue by Vampouille** [georchestra/georchestra#1241]
Datadir overwriten by geoserver_minimal_datadir at each container startup. 

```
Initializing datadir from georchestra/geoserver_minimal_datadir
Archive:  /tmp/datadir.zip
ee204706947ff134e5da28c921e4cfc79b4a320b
replace /tmp/geoserver_minimal_datadir-master/.gitignore? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
(EOF or read error, treating as "[N]one" ...)
```

**Comment by Vampouille** [comment]
`start.sh` script check `.git` directory but this directory is not shipped with this archive :
https://github.com/georchestra/georchestra/blob/master/geoserver/webapp/src/docker/Dockerfile#L4

Fixed with : d27607ff93c54c5a464a9c762e1a236c4bae36f9

**Comment by fvanderbiest** [comment]
Oops, my bad. Sorry !
